tests/NHSD.BuyingCatalogue.Solutions.Persistence.DatabaseTests/SolutionCapabilityRepositoryTests.cs

```diff
@@ -18,9 +20,7 @@ public class SolutionCapabilityRepositoryTests
 
         private const string SupplierId = "Sup 1";
 
-        private readonly Guid _cap1Id = Guid.NewGuid();
-        private readonly Guid _cap2Id = Guid.NewGuid();
-        private readonly Guid _cap3Id = Guid.NewGuid();
+        private List<(Guid id, string name, string desc, string reference)> capDetails;
```

Comment:
This could get very long in the future if we add more properties. Could a class be considered here? It should be specific to this test.